src/DynamicMvvm/Command/DynamicCommand.cs

```diff
@@ -81,7 +84,7 @@ public async Task Execute(object parameter)
 				{
 					IncrementExecutions();
 
-					await _strategy.Execute(_cancellationTokenSource.Token, parameter, this);
+					await _strategy.Execute(_cancellationToken, parameter, this);
```

Comment:
When the command is disposed, we should not even reach that code.
We should just log and error at the very beginning of the Execute method saying that we can't execute a disposed command and return.